src/classes/GE_GiftEntryController.cls

```diff
@@ -114,11 +116,11 @@ public with sharing class GE_GiftEntryController {
     * @return The Id of the new Opportunity record, created from the Data Import
     */
     @AuraEnabled
-    public static Id saveAndProcessDataImport(DataImport__c diRecord, String widgetData, Boolean updateGift) {
+    public static Id saveAndProcessDataImport(DataImport__c diRecord, Boolean updateGift) {
 
         // Check for widget data and convert it to the correct format for processing
-        String additionalObjectJSON = GE_FormRendererService.convertWidgetDataToObjectJSON(widgetData);
-        diRecord.Additional_Object_JSON__c = additionalObjectJSON;
+        diRecord.Additional_Object_JSON__c =
```

Comment:
This works, but we're sending the Additional_Object_JSON__c field to the back-end in a different format from what it is ultimately stored in. I'm not sure what the better way to go about this is, maybe another DI field that accepts the pre-transformation widgetData?

Reply:
Hmm good point. I was trying to reduce the number of things we're passing into apex, but that might preferable to adding an additional field to the DI object.